specification/cognitiveservices/Language.AnalyzeText/common.tsp

```diff
@@ -0,0 +1,78 @@
+import "@typespec/rest";
+import "@typespec/http";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Language.AnalyzeText;
+
+#suppress "@azure-tools/typespec-azure-core/long-running-polling-operation-required" "template"
+@Foundations.Private.needsRoute
+@post
+@doc("Long running operation template to match Azure Language.")
+op LanguageLongRunningOperation<
```

Comment:
Since you're currently only using this template in 1 place under Language.AnalyzeText you can remove this.  I'll follow up in a comment with the suggested code on the route that was using this template.